cloud-init/integration-nocloudkvm.yaml

```
@@ -80,7 +80,7 @@
       - shell: |
           #!/bin/bash
 
-          retry() (for i in {{0..5}}; do sleep $((8*i**3)) && "${{@:1}}" && break; done)
+          retry() (for i in {0..5}; do sleep $((8*i**3)) && {echo "${@:1}"; "${@:1}";} && break; done)
```

I believe the CI failures are happening because these curly braces are no longer escaped:

```suggestion
          retry() (for i in {{0..5}}; do sleep $((8*i**3)) && {{echo "${{@:1}}"; "${{@:1}}";}} && break; done)
```

---

(I'll let you change the other instances of this, rather than proposing a suggestion for each one.)

---

Right, so all places using retry() need the double curly due to JJB expansion.